src/test/java/fi/metatavu/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -83,23 +266,49 @@ private ServiceChannelType resolveChannelType(String type) {
         return null;
     }
   }
+  
+  private String printOrganizationServicesList(int index, File organizationFile) throws IOException {
+    ObjectMapper objectMapper = getObjectMapper();
+    V8VmOpenApiOrganization organization = objectMapper.readValue(organizationFile, V8VmOpenApiOrganization.class);
+
+    StringBuilder result = new StringBuilder();
+
+    result.append("      {\n");
 
+    List<String> ids = organization.getServices().stream()
+      .map(V5VmOpenApiOrganizationService::getService)
+      .map(VmOpenApiItem::getId)
+      .map(UUID::toString)
+      .map(id -> String.format("        \"%s\"", id))
+      .collect(Collectors.toList());
+
+    result.append(StringUtils.join(ids, ",\n"));  
+    result.append("\n      }");
+
+    return result.toString();
+  };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)